src/main/java/com/google/devtools/build/lib/runtime/UiEventHandler.java

```diff
@@ -652,7 +652,7 @@ public void noBuildFinished(NoBuildRequestFinishedEvent event) {
   @Subscribe
   public void afterCommand(AfterCommandEvent event) {
     synchronized (this) {
-      buildRunning = true;
+      buildRunning = false;
     }
     completeBuild();
```

Comment:
From the implementation of `completeBuild`, it looks like this change is exactly equivalent to removing this call.

Reply:
I came to the same conclusion yesterday, but then thought this would be the minimal change to fix the bug in case I didn't spot all the things.